standalone-metastore/metastore-server/src/main/sql/derby/hive-schema-4.0.0-beta-1.derby.sql

```diff
@@ -306,6 +306,8 @@ CREATE INDEX "APP"."CONSTRAINTS_PARENT_TBL_ID_INDEX" ON "APP"."KEY_CONSTRAINTS"(
 
 CREATE INDEX "APP"."CONSTRAINTS_CONSTRAINT_TYPE_INDEX" ON "APP"."KEY_CONSTRAINTS"("CONSTRAINT_TYPE");
 
+CREATE INDEX "APP"."PARTITION_KEY_VALS_IDX" ON "APP"."PARTITION_KEY_VALS"("PART_KEY_VAL");
```

Comment:
If we change the latest version schema, we also should make sure the privious schema to upgrade smoothly.
That is to say, we need to add this change in `upgrade-4.0.0-alpha-2-to-4.0.0-beta-1.derby.sql`.

Maybe you can refer to HIVE-19605 to add the change.